Prox/Prox/PlaceDetails/PlaceDetailViewController.swift

```diff
@@ -198,6 +198,17 @@ class PlaceDetailViewController: UIViewController {
     fileprivate func setBackgroundImage(toPhotoAtURL photoURLString: String?) {
         if let imageURLString = photoURLString,
             let imageURL = URL(string: imageURLString) {
+
+            let imageRequest = URLRequest(url: imageURL)
+            let cachedImage = UIImageView.sharedImageDownloader().imageCache?.imageforRequest(imageRequest, withAdditionalIdentifier: nil)
+
+            // Perform a cross fade between the existing/new images
+            let crossFade = CABasicAnimation(keyPath: "contents")
+            crossFade.duration = 0.4
+            crossFade.fromValue = backgroundImage.image?.cgImage
+            crossFade.toValue = cachedImage
+            backgroundImage.layer.add(crossFade, forKey: "animateContents")
+
             self.backgroundImage.setImageWith(imageURL)
```

Comment:
`cachedImage` will usually return nil when navigating to new images since we haven't fetched them yet so in these cases we'll want to call `setImageWith(url:)` to fetch+load them. If we've already fetched them, we'll just hit the cache so it's the same assigning the image directly.